### PR TITLE
Issue1479 Chiller staging subsequences: Efficiency and failsafe condition

### LINKED ISSUE #1479: Chiller staging: efficiency and failsafe condition
Implement according to the staging table, as provided in RP1711 Draft 4, 5.2.4.14.

### PR DESCRIPTION
**Pull request by milicag**
Closes #1479.

Review after merging [PR#1454](https://github.com/lbl-srg/modelica-buildings/pull/1454) and [PR#1477](https://github.com/lbl-srg/modelica-buildings/pull/1477).

### CODE REVIEW

**Comment by milicag** [comment]
@AntoineGautier I plan to run pre-review tests as soon as we merge [conf. and status PR](https://github.com/lbl-srg/modelica-buildings/pull/1454).

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Validation/FailsafeCondition.mo [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    final amplitude=6895,
+    final offset=63*6895,
+    final freqHz=1/1500,
+    final startTime=0,
+    phase=0.78539816339745)
+    "Chilled water differential pressure"
+    annotation (Placement(transformation(extent={{80,-60},{100,-40}})));
+
+equation
+
```
Comment (milicag) [inline Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Validation/FailsafeCondition.mo]: remove empty line

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Validation/EfficiencyCondition.mo [hunk 2]
```diff
@@ -0,0 +1,57 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Subsequences.Validation;
+model EfficiencyCondition "Validate efficiency condition sequence"
+
+  Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Subsequences.EfficiencyCondition
+    effCon
+    "Failsafe condition to test for the operating part load ratio input"
```
Comment (milicag) [inline Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Validation/EfficiencyCondition.mo]: move to previous line

**Comment by milicag** [comment]
@AntoineGautier This is ready for review as soon as the tests pass.

**Comment by milicag** [comment]
@JayHuLBL This is ready for review.

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/EfficiencyCondition.mo [hunk 3]
```diff
@@ -0,0 +1,81 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Subsequences;
+block EfficiencyCondition
+  "Efficiency condition used in staging up and down"
+
+  parameter Modelica.SIunits.Time delayStaCha = 300
+  "Enable delay";
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/EfficiencyCondition.mo]: should it be 15 minutes (900 s)?

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/EfficiencyCondition.mo [hunk 4]
```diff
@@ -0,0 +1,82 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Subsequences;
+block EfficiencyCondition
+  "Efficiency condition used in staging up and down"
+
+  parameter Modelica.SIunits.Time delayStaCha = 300  "Enable delay";
+
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/EfficiencyCondition.mo]: should it be 15 minutes?

**Comment by JayHuLBL** [comment]
@milicag checked the latest RP1711 and there is no update on this sequence. Besides the one I marked, I have no other comment. 

It should be ready to merge after addressing the comment and CI tests passed.

File: Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/EfficiencyCondition.mo (hunk 4 above)
Comment (milicag) [reply Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/EfficiencyCondition.mo]: yes, corrected.

**Comment by milicag** [comment]
@JayHuLBL I addressed your comment. In the header of the staging table for the newest version there is a conditional for series chillers plants, so I added that in the failsafe sequence and tests. I also added in the info section the 1711 version. When the tests pass please take a quick look before merge due to the added conditional.

**Pull request merged**